roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -976,13 +979,55 @@ RooAbsReal* RooAbsPdf::createNLL(RooAbsData& data, const RooLinkedList& cmdList)
   pc.defineDouble("IntegrateBins", "IntegrateBins", 0, -1.);
   pc.defineMutex("Range","RangeWithName") ;
   pc.defineMutex("GlobalObservables","GlobalObservablesTag") ;
+  pc.defineInt("NewStyle", "NewStyle", 0, 0);
+  pc.defineMutex(
+    "NewStyle",
+    "NumCPU"); // New style likelihoods define parallelization through Parallel(...) on RooMinimizer or fitTo.
+
 
   // Process and check varargs
   pc.process(cmdList) ;
   if (!pc.ok(true)) {
     return 0 ;
   }
 
+  if (pc.getInt("NewStyle")) {
+      int lut[3] = {2, 1, 0};
+      RooFit::TestStatistics::RooAbsL::Extended ext{
+        static_cast<RooFit::TestStatistics::RooAbsL::Extended>(lut[pc.getInt("ext")])};
+
+      RooArgSet cParsSet;
+      RooArgSet extConsSet;
+      RooArgSet glObsSet;
+
+      auto tmp = pc.getSet("cPars");
+      if (tmp)
+        cParsSet.add(*tmp);
```

Comment:
```suggestion
      if (auto tmp = pc.getSet("cPars");)
        cParsSet.add(*tmp);
```
You can make the code mode compact if you do this here and also in the lines below